src/components/PostComment.jsx

```diff
@@ -0,0 +1,55 @@
+import { postComments } from "../../api";
+import { useState } from 'react';
+
+const PostComment = ({article_id}) => {
+    const [loading, setLoading] = useState(false)
+    const [error, setError] = useState(null)
+    const [commentBody, setCommentBody] = useState("")
+    const [username, setUsername] = useState("")
+
+
+const handleUsernameChange = (event) => {
+    setUsername(event.target.value)
+}
+ 
+const handleCommentChange = (event) => {
+    setCommentBody(event.target.value)
+}
+
+const handlePostComment = () => {
+        setLoading(true)
+        postComments(article_id, username, commentBody)
+        .then((response) => {
+            console.log("your comment has successfully been posted")
+        })
+    .catch ((error) => {
+        setError(error.message)
+        setLoading(false)
+    })
+}
+
+return (
+    <div>
+        <h2>Leave a comment</h2>
+        <div className ="addCommentContainer">
+            <label htmlFor="comment">Your Comment:</label>
+            <textarea 
+            commentsection="comment" 
+            value={commentBody} 
+            onChange={handleCommentChange}
+            ></textarea>
+            <label htmlFor="username">Your Username:</label>
+            <input type="text" 
+            username="username"
+            value={username}
+            onChange={handleUsernameChange}
```

Comment:
It's a bit odd to ask the user for their username when submitting the form. Usually the app would already know who the user is. I would store are hardcoded user as a variable and use that.